hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirectory.java

```diff
@@ -1543,6 +1574,43 @@ public final void addRootDirToEncryptionZone(XAttrFeature xaf) {
     addEncryptionZone(rootDir, xaf);
   }
 
+  /**
+   * After the inodes are set properly (set the parent for each inode), we move
+   * them from INodeMapTemp to INodeMap.
+   */
+  public void moveInodes() throws IOException {
```

Comment:
Doesn't need to be public. Package private would do?
Please also check other use of `public`. The most restrictive modifier should be used.

Reply:
fixed and also made other functions package private.